codegen/facelift-codegen.py

```diff
@@ -225,6 +236,10 @@ def requiredInclude(self):
 
 setattr(qface.idl.domain.Module, 'namespaceCppOpen', property(namespaceCppOpen))
 setattr(qface.idl.domain.Module, 'namespaceCppClose', property(namespaceCppClose))
+#setattr(qface.idl.domain.Module, 'classExportDefines', classExportDefines)
+#setattr(qface.idl.domain.Module, 'classExport', classExport)
```

Comment:
If that is really not needed it should be removed.

Reply:
done